content/docs/user-guide/gems/reference/network/http-requestor.md

```diff
@@ -25,6 +25,29 @@ BUILD_DEPENDENCIES
             3rdParty::AWSNativeSDK::Core
 ```
 
+### Create a dependency on the Gem
+
+If you need to ensure the HttpRequestor Gem is enabled before its use in a component, add a requirement in your component's `GetRequiredServices` method. For example:
+
+```cpp
+void AutomatedTestingSystemComponent::GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
+{
+   ...
+   required.push_back(AZ_CRC_CE("HttpRequestorService"));
+}
+```
+
+### Turn off Amazon EC2 Instance Metadata Service calls
+
+The HttpRequestor Gem uses the [AWS C++ SDK](https://github.com/aws/aws-sdk-cpp) to provide the Http(s) client. You don't need to provide AWS credentials or account information to use this Gem.
+
+However, if your project is not running on Amazon EC2 compute, it's recommended that you turn off the [AWS_EC2_METADATA_DISABLED](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) environment variable. This will prevent any calls to the Amazon EC2 Instance Metadata Service (IMDS) from attempting to retrieve configuration, region, and credential information. Requests to IMDS will fail if you are not using EC2, leading to delays and wasted network resources.
```

Comment:
I know this comment is late and a bit of a nitpick but it should probably be phrased as "turn _on_ the AWS_EC2_METADATA_DISABLED environmental variable..." since it gets set to `true`.